bft/testers/cudf/runner.py

```diff
@@ -1,11 +1,29 @@
-from typing import Dict, NamedTuple
-
 import cudf
+import math
 
 from bft.cases.runner import SqlCaseResult, SqlCaseRunner
 from bft.cases.types import Case, CaseLiteral
 from bft.dialects.types import SqlMapping
 
+unary_ops = {
+    "sqrt",
+    "exp",
+    "sin",
+    "cos",
+    "tan",
+    "cosh",
+    "sinh",
+    "tanh",
+    "arccos",
+    "arcsin",
+    "arctan",
+    "arccosh",
+    "arcsinh",
+    "arctanh",
+    "~",
```

Comment:
If this is an operator like plus and minus should it be treated similarly?

Reply:
There is a special case of functions called [unary operators](https://docs.rapids.ai/api/libcudf/stable/group__transformation__unaryops.html) that are evaluated against cudf.Dataframes objects, whereas all the other functions are evaluated against cudf.Series objects.  But your comment made me realize I don't actually need this list